### PR TITLE
Instructions for node and sql project review

### LINKED ISSUE #92: Create "##Instructions for reviewer" for project: Node and SQL assignment
Please add a section to the bottom of this project's _index.md file.
The section heading should be EXACTLY "## Instructions for reviewer"

Basically we have to set reviewers up for success by telling them what success looks like and what to pay attention to when reviewing this project submission.

Description:

- [ ] Look at a bunch of old reviews for this project for each cohort

- [ ] Find the common feedback [This should help you get an idea of what instructions to create]

- [ ] Write them out clearly

- [ ] The section heading should be EXACTLY "## Instructions for reviewer"

### PR DESCRIPTION
**Pull request by Sbonelo01**
Related issues: [#92 ]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/projects/nodejs/sql/_index.md [hunk 1]
```diff
@@ -59,4 +59,20 @@ You will be expected to properly test your code. You can use whatever testing fr
...
+
+- Connection strings should never be exposed. Ensure that the `.dotenv` module has been utilized.
+
+- The `.dotenv` file should exist in the `.gitignore` file.
+
+- Ensure that `Docker` and `docker-compose` have been used. The `docker-compose.yml` file should be at the root of the project.
+
+- Ensure that the proper data type is returned for each function.
+
+- Ensure that `Pool.connect || Client.connect` and `Pool.end || Client.end` is used correctly. there is generally no need to for opening and closing your connection inside every function.
```
Comment (ry-oc) [inline content/projects/nodejs/sql/_index.md]: Capitalise `There` at the start of the new sentence

File: content/projects/nodejs/sql/_index.md [hunk 2]
```diff
@@ -59,4 +59,20 @@ You will be expected to properly test your code. You can use whatever testing fr
...
+
+- Ensure the use of `return`, not `console.log`.
+
+- When it comes to testing for this project, have a look at {{% contentlink path="topics/unit-testing-mocks-and-spies" %}} (jasmine specific), and make sure that mocks and spies are utilized properly.
```
Comment (ry-oc) [inline content/projects/nodejs/sql/_index.md]: Isn't this topic {{% contentlink path="topics/unit-testing-mocks-and-spies" %}}  language agnostic? I can see some python stuff there too.

I think just remove `(jasmine specific)`

File: content/projects/nodejs/sql/_index.md [hunk 3]
```diff
@@ -59,4 +59,20 @@ You will be expected to properly test your code. You can use whatever testing fr
...
 - https://www.guru99.com/introduction-to-database-sql.html
-- https://www.w3schools.com/sql/sql_intro.asp
\ No newline at end of file
+- https://www.w3schools.com/sql/sql_intro.asp
+
+## Instructions for reviewers
+
+- Connection strings should never be exposed. Ensure that the `.dotenv` module has been utilized.
+
+- The `.dotenv` file should exist in the `.gitignore` file.
```
Comment (ry-oc) [inline content/projects/nodejs/sql/_index.md]: Could we maybe reword this line. `The .dotenv file should not be present in the repo and should be specifically excluded through use of .gitignore.` I think that make more sense but correct me if I am wrong?

**Review by ry-oc — CHANGES_REQUESTED** [review]
Smallinyana changes

File: content/projects/nodejs/sql/_index.md (hunk 2 above)
Comment (Sbonelo01) [reply content/projects/nodejs/sql/_index.md]: The reason I included `(jasmine specific)` is because the learners are given an opportunity to choose the testing framework of their choice for this project @ry-oc

**Review by ry-oc — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]
